src/views/TradingCompetition/components/RibbonWithImage/index.tsx

```diff
@@ -18,7 +18,9 @@ const Spacer = styled.div`
 `
 
 const AbsoluteImageWrapper = styled.div`
+  z-index: 2;
   position: absolute;
+  bottom: -8px;
```

Comment:
Can you add a comment here why it is -8px

Reply:
👍